### PR TITLE
20.9.0 Release Notes

### PR DESCRIPTION
**Pull request by hamdiallam**
Release notes blog post for 20.9.0

### CODE REVIEW

File: source/blog/2020-09-24-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+layout: post
+title: September 2020 Release Notes - Version 20.9.0
+published: true
+post_author:
+  display_name: Hamdi Allam
+  twitter: allam_hamdi
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+With the start of Fall, it's only right for a new release of our libraries. Here's the latest of Finagle, Finatra, Scrooge, and Util!
```
Comment (yufangong) [inline source/blog/2020-09-24-release-notes.md]: Missing Twitter-Server entries?

File: source/blog/2020-09-24-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+### Finagle
+
+New Features
```
Comment (yufangong) [inline source/blog/2020-09-24-release-notes.md]: Lets do ```#### New Features```

**Review by yufangong — APPROVED** [review]

File: source/blog/2020-09-24-release-notes.md (hunk 1 above)
Comment (hamdiallam) [reply source/blog/2020-09-24-release-notes.md]: Just pushed a commit that fixed this

File: source/blog/2020-09-24-release-notes.md (hunk 2 above)
Comment (hamdiallam) [reply source/blog/2020-09-24-release-notes.md]: same as above

File: source/blog/2020-09-25-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: September 2020 Release Notes - Version 20.9.0
```
Comment (cacoco) [inline source/blog/2020-09-25-release-notes.md:3]: What, no emoji?

Comment (hamdiallam) [reply source/blog/2020-09-25-release-notes.md]: forgot the ctrl + cmd + space shortcut. Emojified 🏀

**Pull request merged**